classes/DwcArchiverResourceRelationship.php

```diff
@@ -0,0 +1,134 @@
+<?php
+include_once($SERVER_ROOT . '/classes/DwcArchiverBaseManager.php');
+
+class DwcArchiverResourceRelationship extends DwcArchiverBaseManager{
+    public function __construct($connOverride){
+		parent::__construct('write', $connOverride);
+	}
+
+	public function __destruct(){
+		parent::__destruct();
+	}
+
+	public function initiateProcess($filePath){
+		$this->setFieldArr();
+		$this->setSqlBase();
+
+
+		$this->setFileHandler($filePath);
+	}
+
+    //Based on https://rs.gbif.org/extension/resource_relationship_2024-02-19.xml
+	private function setFieldArr(){
+		$columnArr = array();
+		$termArr['deleteMeOccid'] = 'https://dwc.tdwg.org/terms/#dwc:resourceRelationshipID';
+		$columnArr['deleteMeOccid'] = 'oa.occid';
+		$termArr['deleteMeOccidAssociate'] = 'https://dwc.tdwg.org/terms/#dwc:resourceRelationshipID';
+		$columnArr['deleteMeOccidAssociate'] = 'oa.occidAssociate';
+		$termArr['resourceRelationshipID'] = 'https://dwc.tdwg.org/terms/#dwc:resourceRelationshipID';
+		// $columnArr['resourceRelationshipID'] = 'IFNULL(IFNULL(IFNULL(oa.objectID, o.occurrenceID), o.recordID), oa.resourceUrl)';
```

Comment:
I'll delete some of these commented-out lines once we're more confident about the current strategy.